src/ui.js

```diff
@@ -169,44 +205,44 @@ export function buildSettingsModal(tracks, opts, finishCallback) {
     })
 
     let modalContent = `
-<h3>Options</h3>
-
-<form id="settings">
-    <span class="form-row">
-        <label>Theme</label>
-        <select name="theme">
-            ${themes}
-        </select>
-    </span>
-
-    <span class="form-row">
-        <label>Line color</label>
-        <input name="color" type="color" value=${opts.lineOptions.color}>
-    </span>
-
-    <span class="form-row">
-        <label>Line opacity</label>
-        <input name="opacity" type="range" min=0 max=1 step=0.01
-            value=${opts.lineOptions.opacity}>
-    </span>
-
-    <span class="form-row">
-        <label>Line width</label>
-        <input name="weight" type="number" min=1 max=100
-            value=${opts.lineOptions.weight}>
-    </span>
-
-    <span class="form-row">
-        <label>Override existing tracks</label>
-        <input name="overrideExisting" type="checkbox" ${overrideExisting}>
-    </span>
-
-    <span class="form-row">
-        <label>Detect color from Strava bulk export</label>
-        <input name="detectColors" type="checkbox" ${detect}>
-    </span>
-</form>
-`;
+        <h3>Options</h3>
```

Comment:
Could we keep the indentation here the same as it was before?

Reply:
443e466e3513891ef4bff9c5029d4e63c64d09ea